include/onnxruntime/core/graph/graph.h

```diff
@@ -941,6 +941,15 @@ class Graph {
   const ONNX_NAMESPACE::GraphProto& ToGraphProto();
   ONNX_NAMESPACE::GraphProto ToGraphProto() const;
 
+  /** Gets the GraphProto representation of this Graph
+  @params external_file_name name of the binary file to use for initializers
+  @param initializer_size_threshold initializers larger or equal to this threshold (in bytes) are saved
+  in the external file. Initializer smaller than this threshold are included in the onnx file.
+  @returns GraphProto serialization of the graph.
+  */
+  ONNX_NAMESPACE::GraphProto ToGraphProtoWithExternalInitializers(const std::string& external_file_name,
+                                                                  size_t initializer_size_threshold) const;
```

Comment:
>initializer_size_threshold [](start = 73, length = 26)

nit: since this value is never changed, please consider changing this variable to const. (same for the other functions where the threshold size is passed)

Reply:
fwiw it's passed by value, so const or not doesn't really matter.

Reply:
Right, other functions in graph.h don't mark by-value parameters as const, so I would it keep it this way.